doc/man/man3/crypto_chacha20_encrypt.3monocypher

```diff
@@ -57,8 +57,8 @@ It needs a 32-byte secret
 and an 8-byte
 .Fa nonce .
 The nonce must be used only once per secret key (repeating them
-destroys confidentiality).  Counters and (unique) message numbers can
-be used as nonces.
+destroys confidentiality).  Counters and unique message numbers can be
```

Comment:
Oops, I missed a dot in the middle of the line there.

Reply:
fixed, grepped for "\. " and found no other instances